install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -36,6 +36,7 @@ spec:
             description: BrokerSpec defines the desired state of Broker
             properties:
               size:
+                description: Desired size of the Broker
```

Comment:
"size" in terms of number of instances or amount of RAM or CPU cores?

Reply:
I have inputted a better description for the "size".